core/test/cljfmt/format/core_test.clj

```diff
@@ -172,7 +183,7 @@
     (is (= "#inst\n\"2018-01-01T00:00:00.000-00:00\""
            (reformat-string "#inst\n\"2018-01-01T00:00:00.000-00:00\""))))
   (testing "map prefixes"
-    (is (= "#:abc\n{:d 1}"
+    (is (= "#:abc\n {:d 1}"
```

Comment:
Could you move this next to the other map tests? This also doesn't seem right, it's saying that this would be indented like:
```clojure
#:abc
 {:d 1}
```
The map body should be aligned with the tag if there's a line-break.

Reply:
The map body is aligned with the tag. :^)

More seriously, there are three separate arguments here I'll make:
1. I'm not convinced that this format is wrong. Philosophically, if you treat the `#` as opening a `:namespaced-map` form, it makes sense that the form body should be indented to match (like how you indent a normal `:map` form: `{:a 1\n :b 2}`).
2. This is how the weavejester implementation formats this. I'm not sure how big a goal compatibility is to you.
3. I don't think we have enough information with the current data structures to fix this problem. For multiline inner maps, to format them "correctly", you would need to check to see if you're also in a `:namespaced-map`, and also if the first line of the `:namespaced-map` is inline or on a new line, and then only indent in the first case.

Reply:
Happy to chat in person, also.

Reply:
Maybe we could just do the change to `core.clj`? That would fix the indentation for abutting maps without screwing up indentation to maps that don't have the first element on the same line as the keyword.

Reply:
Fixed.